listener/message.go

```diff
@@ -14,14 +14,14 @@ import (
 func newHubMessage(payload []byte, protocol, uri string) hub.Message {
 
 	host, err := os.Hostname()
-
 	if err != nil {
 		host = "UNKNOWN"
 	}
 
 	m := hub.Message{
 		Payload: payload,
 		Metadata: map[string]interface{}{
+			hub.SHA256_SUM:         "",
```

Comment:
Could you change this to a better default value e.g. something like "not calculated"  or "pending" ?